### PR TITLE
Dom tests

### PR DESCRIPTION
**Pull request by blink1073**
```
----------------|----------|----------|----------|----------|----------------|
File            |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
----------------|----------|----------|----------|----------|----------------|
 dom/           |    93.33 |    82.54 |       80 |    93.33 |                |
  cursor.js     |      100 |      100 |      100 |      100 |                |
  query.js      |      100 |      100 |      100 |      100 |                |
  selector.js   |    90.36 |    64.52 |    66.67 |    90.36 |... 164,165,231 |
  sizing.js     |      100 |      100 |      100 |      100 |                |
----------------|----------|----------|----------|----------|----------------|
All files       |    98.94 |    93.33 |    98.82 |    99.26 |                |
----------------|----------|----------|----------|----------|----------------|
```

### CODE REVIEW

**Comment by blink1073** [comment]
I didn't write tests for `platform.ts` because it would depend on which platform you were using ;)

File: test/src/dom/cursor.spec.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) 2014-2015, PhosphorJS Contributors
+|
+| Distributed under the terms of the BSD 3-Clause License.
+|
+| The full license is in the file LICENSE, distributed with this software.
+|----------------------------------------------------------------------------*/
+'use strict';
```
Comment (sccolbert) [inline test/src/dom/cursor.spec.ts]: 'use strict' is no longer required as of TS 1.8 - it gets added automatically by the compiler

File: test/src/dom/query.spec.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) 2014-2015, PhosphorJS Contributors
+|
+| Distributed under the terms of the BSD 3-Clause License.
+|
+| The full license is in the file LICENSE, distributed with this software.
+|----------------------------------------------------------------------------*/
+'use strict';
```
Comment (sccolbert) [inline test/src/dom/query.spec.ts]: 'use strict' is no longer required as of TS 1.8 - it gets added automatically by the compiler

File: test/src/dom/sizing.spec.ts [hunk 3]
```diff
@@ -0,0 +1,87 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) 2014-2015, PhosphorJS Contributors
+|
+| Distributed under the terms of the BSD 3-Clause License.
+|
+| The full license is in the file LICENSE, distributed with this software.
+|----------------------------------------------------------------------------*/
+'use strict';
```
Comment (sccolbert) [inline test/src/dom/sizing.spec.ts]: 'use strict' is no longer required as of TS 1.8 - it gets added automatically by the compiler

File: test/src/dom/cursor.spec.ts [hunk 4]
```diff
@@ -0,0 +1,91 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) 2014-2015, PhosphorJS Contributors
```
Comment (sccolbert) [inline test/src/dom/cursor.spec.ts]: 2014-2016

File: test/src/dom/query.spec.ts [hunk 5]
```diff
@@ -0,0 +1,53 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) 2014-2015, PhosphorJS Contributors
```
Comment (sccolbert) [inline test/src/dom/query.spec.ts]: 2014-2016

File: test/src/dom/query.spec.ts [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+      expect(hitTest(div, 0, 100)).to.be(false);
+      expect(hitTest(div, 0, 99)).to.be(true);
+      expect(hitTest(div, 100, 100)).to.be(false);
+      expect(hitTest(div, 99, 99)).to.be(true);
+      document.body.removeChild(div);
+    });
+
+  });
+
+});
```
Comment (sccolbert) [inline test/src/dom/query.spec.ts]: can you add the test for `scrollIfNeeded()`, which was just added to this module?

File: test/src/dom/sizing.spec.ts [hunk 7]
```diff
@@ -0,0 +1,87 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) 2014-2015, PhosphorJS Contributors
```
Comment (sccolbert) [inline test/src/dom/sizing.spec.ts]: 2014-2016

**Comment by blink1073** [comment]
Comments addressed.  Also moved test css to a single file because the whole test suite gets bundled anyway.

**Pull request merged**